### ISSUE TITLE
[idea] easy-peasy-core without the react binding to create redux store in other framework

### ISSUE THREAD

**Issue by stevenao** [ctrlplusb/easy-peasy#232]
Love the easy workflow this library providing. I would like to use it in other framework like polymer and LitElement. Since the store this library is generating pretty much just a redux store, I see a high chance of compatibility. The architecture question would be that if it is better to have a `easy-peasy-core` and a `easy-peasy-react` hooks binding library. Maybe `easy-easy-angular` too?

That way I can just import the `core` library without the react stuff when I use it in the Polymer/LitElement framework.

**Comment by ctrlplusb** [comment]
Hey @stevenao - thanks for your interest.

To be honest I am unlikely to ever do this, unless I personally start using Polymer or other frameworks.

The primary reason for my open source work is to satisfy problems that I myself am experiencing. This keeps me motivated to maintain it and ensures I am empathetic to the users.

I would be completely cool with you creating an `easy-peasy-core` fork of this library though. I think it would be quite simple to remove the React references and then build and publish the library. I can even publish a link to your fork within the website. 👍

**Comment by CyriacBr** [comment]
Hello @ctrlplusb , I came across the same needs as stevano. I work on a framework-agnostic monorepo project and I very much would like to use EP.
Would you be ok if I publish some forks of `easy-peasy` for each version (`core`, `react`, `vuejs` and `angular`) under a npm scope...maybe `easy-peasy-standalone`?

**Comment by ctrlplusb** [comment]
👍

Completely cool with this. I can even link you from the official site.

**Comment by CyriacBr** [comment]
Oh alright if `@easy-peasy` is available I'll take this one then. I just didn't want to shadow your work. Thanks for answering this quickly. I'll get back once I have the repo set up.

**Comment by ctrlplusb** [comment]
No problem. Thanks for checking with me. 💜

**Comment by CyriacBr** [comment]
@ctrlplusb Here's the repo: https://github.com/CyriacBr/easy-peasy-packages.
I published `@easy-peasy/core` and `@easy-peasy/react`.
Respective tests pass so far. I've used core in my project just now and all seem good. 

Btw, I tried to convert the react part of your lib in typescript but wow, it wasn't an easy task.  I gave up at a point. When I'll work on the vue connector I'll see if I can correctly leverage your types to provide a full TS codebase.
